### DISCUSSION TITLE
DataView Property

### DISCUSSION THREAD

**Discussion by Rasenkantenstein** [TabularEditor/TabularEditor3#1143] [🙋♀️TE3 Q&A] [answered]
Hi TE-Team,

I have a question regarding this property:
![image](https://user-images.githubusercontent.com/6643712/197701754-c32e4700-6d76-490f-b16f-c628c43c5a62.png)

Does TabularEditor still use this property? When clicking on "update table scheme", which partition is selected to run the schema-compare?

Best regards,
Konstantin

**Comment by otykier** [comment, marked as answer]
Hi Konstantin
TE simply exposes all TOM properties. At some point, we may provide a toggle option for the Properties window, to only show the most commonly used properties (a kind of “Simple mode”). As such, the Data View property does not impact how TE3 displays data or performs schema updates. The latter are always inferred from the first partition (in metadata order) on the table.
